contact-track/src/App.js

```diff
@@ -0,0 +1,129 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+import {
+  Table,
+  TableBody,
+  TableHeader,
+  TableHeaderColumn,
+  TableRow,
+  TableRowColumn,
+} from 'material-ui/Table';
+
+import ReactTable from 'react-table'
+import 'react-table/react-table.css'
+import FlatButton from 'material-ui/FlatButton';
+
+var ReactTable = require('react-table').default
+
+const FlatButtonExampleSimple = () => (
+  <div>
+    <FlatButton label="Submit" primary={true} />
+    <br />
+    <br />
+  </div>
+);
+
+//need to set onclick for button that allows user to submit new info to table (change state fn)
+//i.e. onclick in render. :
+/* render() {
```

Comment:
There should never be a need to commit dead (commented out) code. The beauty of git is that you can always revert